### DISCUSSION TITLE
why  Measurements can contain up to 200 columns  in cloud-serverless

### DISCUSSION THREAD

**Discussion by cqzhang7** [influxdata/influxdb#25053] [Q&A]
As the title says，I don't understand why there is a mandatory limit on the number of measurement columns. Can someone answer this? Thanks.

**Comment by pauldix** [comment]
Wide tables can cause problems in the InfluxDB 3 engine. Both in dealing with the underlying Parquet files and in other areas in the system. This can be particularly problematic in a muti-tenant system like Serverless so we enforce that limit.
